schema/dataKeys.json

```diff
@@ -137,7 +137,7 @@
         "product_version": {
           "type": "string",
           "description": "The product version e.g v1.0.0",
-          "pattern": "^fbc|(fbc[-]|[vV])?(?:0|[1-9]\\d*)\\.(?:0|[1-9]\\d*)(?:\\.(?:0|[1-9]\\d*))?([-\\s]?([Aa]lpha|[Bb]eta|fast|tech[-\\s]preview)?)$"
+          "pattern": "^(fbc|((fbc[-]|[vV])?(?:0|[1-9]\\d*)(?:\\.(?:0|[1-9]\\d*)){0,2}([-\\s]?([Aa]lpha|[Bb]eta|fast|tech[-\\s]preview))?))$"
```

Comment:
Based on the discussion, do we remove this check now or will that be done in another PR?